src/Theme.tsx

```diff
@@ -0,0 +1,31 @@
+import { ThemeProvider } from "styled-components";
+
+const theme = {
+  colors: {
+    offWhite: "#dfefff",
+    darkGrey: "#333333",
+    primaryBlue: "#3988ff",
+    skyBlue: "#B2DAFF",
+    cloudBlue: "#E5F2FF",
+  },
+  fonts: {
+    heading: "Poppins, sans-serif",
+    content: "Source Sans Pro, sans-serif"
+  },
+  fontSizes: {
+    small: "1em",
+    medium: "2em",
+    large: "3em"
+  },
+  viewport: {
+    mobile: "(min-width: 425px)",
```

Comment:
I think this should be 320px since no styling will apply on an iPhone X w/ this min-width here